### ISSUE TITLE
a problem about the final accuracy

### ISSUE THREAD

**Issue by abolfazlzamanii** [angus924/rocket#12]
Hello, I am a master's student and I am working on your article. I have a problem about the final accuracy, even though I actually implemented your code, I get much less accuracy. I have put the link related to the code below. If you tell me the error of the code, I will be grateful. 

https://colab.research.google.com/drive/1vH9qAI2lgF_0htYPhU8UQb45aGUBEGk5?usp=share_link

**Comment by angus924** [comment]
Hi @abolfazlzamanii, thanks for your message.

I have had a quick look at your code, but I have not run it.

It looks like there might be a couple of issues:
1. The range of regularization (alpha) values for the ridge classifier doesn't look right: try `alphas = np.logspace(-3, 3, 10)`, that's the default setting we use in all our code.
2. Rocket is not intended to be used with the `StandardScaler`, although it shouldn't make too much difference.  In any case, I notice that you're fitting the StandardScaler to both the training and test data, which doesn't make sense.  The StandardScaler should be fit to the training data, and then used to transform both the training and test data, i.e., `scaler.fit_transform(X_training_transform)` and then `scaler.transform(X_test_transform)`.  Note that the StandardScaler subtracts the mean and divides by the standard deviation, whereas we subtract the mean and divide by the L2 norm (which is the original behaviour of the now-deprecated `normalize = True` argument of the ridge classifier class).

Let me know if you're still having issues and we can try to sort them out.

Thanks.
